docker/logging_server/source/src/main/java/jp/co/yahoo/yosegi/tools/server/logger/FluentdLogger.java

```diff
@@ -0,0 +1,34 @@
+package jp.co.yahoo.yosegi.tools.logging.server.logger;
+
+import jp.co.yahoo.yosegi.tools.logging.server.config.YamlObjectUtils;
+
+import org.fluentd.logger.FluentLogger;
+
+import java.io.IOException;
+import java.util.Map;
+import java.util.Properties;
+import java.util.concurrent.ExecutionException;
+
+public class FluentdLogger implements ILogger {
+
+  private FluentLogger logger;
+  private String label;
+  private String messageKey;
+
+  @Override
+  public void setup( final Map setting ) throws IOException {
+    String tag = YamlObjectUtils.getString( setting.get( "tag" ) , "default" );
+    String host = YamlObjectUtils.getString( setting.get( "host" ) , "localhost" );
+    int port = YamlObjectUtils.getInt( setting.get( "port" ) , 24224 );
```

Comment:
24224 is the default of the port of fluentd, is it right?

Reply:
Yes, it is the default port for fluentd.

Reply:
Thank you for your consideration. I check the fixation later and approve this PR.